src/hdf5/apply_BG/apply_BG.py

```diff
@@ -0,0 +1,181 @@
+"""
+J. Zhou, H. Yang and R. Iacovazzi,
+"Improving ATMS Remapping Accuracy Using Adaptive Window and Noise-Tuning Method in Backus–Gilbert Inversion,"
+in IEEE Transactions on Geoscience and Remote Sensing,
+vol. 60, pp. 1-12, 2022, Art no. 5304412, doi: 10.1109/TGRS.2022.3182630.
+
+Original source code written by Jun Zhou @UMD can be found at:
+https://github.com/JunUMD/BG_AdaptiveWindow_v1
+"""
+from pathlib import Path
+import os, glob, time
+import pickle, sys
+import h5py
+import numpy as np
+import statistics
+from scipy.interpolate import griddata
+
+
+class apply_BG_class:
```

Comment:
Is the `apply_BG` functionality something that is specific to ATMS data, or is `apply_BG` something that could be used for many instrument type. I'm wondering if `apply_BG` belongs here (specific to ATMS) or if it possibly belongs in UFO somewhere instead (can apply to many instruments).

Reply:
Sorry for the late reply @srherbener. apply_BG can also be used for other instruments.

Reply:
Sorry for my late reply too. I lost track of this PR.

I agree that it is good to place apply_BG and related functions in a separate python file/module so that this code can be shared. Does it seem that `src/hdf5` is the place for this, or would it make sense to move to `src/lib-python`?

Thanks for taking care of this!